src/main/java/org/reficio/p2/P2Validator.java

```diff
@@ -60,19 +60,19 @@ public static void validateArtifactConfig(P2Artifact p2Artifact, ResolvedArtifac
             // artifact is a bundle and somebody specified instructions without override
             if (!p2Artifact.shouldOverrideManifest() && !p2Artifact.getCombinedInstructions().isEmpty()) {
                 String message = String.format("p2-maven-plugin misconfiguration" +
-                        "\n\n\tJar [%s] is already a bundle. " +
-                        "\n\tBND instructions are specified, but the <override> flag is set to false." +
-                        "\n\tEither remove the instructions or set the <override> flag to true." +
-                        "\n\tWATCH OUT! Setting <override> to true will re-bundle the artifact!\n", resolvedArtifact.getArtifact().toString());
+                        "%n%n\tJar [%s] is already a bundle. " +
+                        "%n\tBND instructions are specified, but the <override> flag is set to false." +
+                        "%n\tEither remove the instructions or set the <override> flag to true." +
+                        "%n\tWATCH OUT! Setting <override> to true will re-bundle the artifact!%n", resolvedArtifact.getArtifact().toString());
                 throw new RuntimeException(message);
             }
             // artifact is a bundle and somebody specified singleton flag without override
             if (!p2Artifact.shouldOverrideManifest() && p2Artifact.isSingleton()) {
                 String message = String.format("p2-maven-plugin misconfiguration" +
```

Comment:
While at it, you should also explicitly define a locale (`String.format(Locale.ENGLISH, "…"`).

Reply:
Will do. Thanks